Reject bad epochs in apply_ica diagnostic plot (#385)
mne-tools/mne-bids-pipeline: docs/source/changes.md

- `epochs_tmin` and `epochs_tmax` were named incorrectly in some test config
  files.
  ({{ gh(340) }} by {{ authors.crsegerie }})

> - We now reject bad epochs by using `ica_reject` before plotting the 

Reviewer: Can you insert a link to the docs here? 
```
[`ica_reject`][config....]
```

Reviewer: Proposal:

"… before producing the "overlay" plots that show the evoked data before and after ICA cleaning."

Author: ok!

> - We now reject bad epochs trespassing `ica_reject` before plotting the 

Reviewer 2: ```suggestion
- We now reject bad epochs by using `ica_reject` before plotting the 
```

>   figures in the `proc-ica_report`.

Reviewer: Haha, nicely phrased, I giggled :)

But I'm not sure if it's easy to understand for someone who wasn't involved in our discussion to understand what has been fixed here. Proposal:

```suggestion
- When creating the "overlay" plot to show the effects of ICA cleaning, we
  previously plotted the data without applying the pre-ICA peak-to-peak
  rejection thresholds, producing misleading figures in some situations. This
  has now been fixed.
```

Reviewer: I see you've meanwhile updated the changelog, so ignore this comment

>   ({{ gh(?) }} by {{ authors.hoechenberger }}), {{ authors.agramfort }} and 
>   {{ authors.crsegerie }})

Reviewer: ```suggestion
  ({{ gh(385) }} by {{ authors.crsegerie }})
```
It's your PR :)
